doc/BENCHMARKING.md

```diff
@@ -0,0 +1,45 @@
+## Benchmarking
+Currently, Mountpoint for Amazon S3 is an alpha release and we are focusing on delivering high throughput read support.  dIn order to make sure that new changes do not introduce performance regression we have created some performance benchmark  das a part of our CI using [fio](https://github.com/axboe/fio), an awesome opensource application for file system benchmarking.
+
+### Workloads
+
+There are two main types of workload we are testing, sequential read and random read. Each of the test is defined in a separate .fio file, and the file name indicates what is the test case for that file, for example `seq_read.fio` is the benchmark for sequential read. All of fio configuration files can be found at path [mountpoint-s3/scripts/fio/](https://github.com/awslabs/mountpoint-s3/tree/main/mountpoint-s3/scripts/fio).
+
+In general, we run each IO operation for 30 seconds against a 100 GiB file. But there are some variants in configuration we also want to test to see how Mountpoint would perform with these configurations. Here is the list of all the variants we have tested.
+
+* **four_threads**: running the workload concurrently by spawning four threads to do the same job.
```

Comment:
Are these FIO threads or FUSE threads?

Reply:
It's FIO threads, added that to the sentence.